tests/core/contracts/test_contract_method_to_argument_matching.py

```diff
@@ -168,22 +169,69 @@ def test_finds_function_with_matching_args_non_strict(
 
 def test_finds_function_with_matching_args_strict_type_checking_by_default(w3):
     contract = w3.eth.contract(abi=MULTIPLE_FUNCTIONS)
-    with pytest.raises(MismatchedABI):
+    with pytest.raises(
+        MismatchedABI,
+        match=re.escape(
+            "\nABI Not Found!\n"
+            "Multiple elements were found matching 1 arguments.\n"
+            "Provided argument types: (str)\n"
+            "Provided keyword argument types: {}\n\n"
+            "Encountered problems with the following elements named `a`.\n"
```

Comment:
It took me a minute to figure out what this error message was telling me. Maybe something along the lines of: `Tried to find a matching ABI Element `a`, but encountered the following problems:` 🤷

Reply:
/cc our resident words guy ™️ @wolovim